doc/nrf/releases_and_maturity/releases/release-notes-2.9.1.rst

```diff
@@ -0,0 +1,87 @@
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF54, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.9.0 <ncs_release_notes_290>`.
+.. todo
+
+See :ref:`ncs_release_notes_291_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.9.1**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.9.1`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.9.1.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.9.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.9.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.9.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_291_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+.. todo
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee``.
```

Comment:
@rlubos, @gmarull - Can you please confirm the commit here?

Reply:
In sdk-nrf v2.9-branch, the west.yml of sdk-zephyr points to ca954a6216c986c1a1d978e9e42ff773a2c7155a ("[nrf fromtree] drivers: clock_control: nrf: Add workaround for XO start anomaly")
And the commit you have here beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee is ("docs: i3c: fix incorrect detail for SETDASA I3C CCC usage") which are not the same. 
The commit ("drivers: clock_control: nrf: Add workaround for XO start anomaly") is 2cb2cf226c0254d662794b8158dd05688eb8e57b in upstream zephyr.

It might be that I am looking the wrong branches.

Reply:
Checked offline with Robert - correct commit is ``beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee`` as cherry-picks don't count.

Reply:
@jukkar I think the SHA was correct here, note that the docs say "**all commits** from the upstream Zephyr repository **up to**", so it only makes sense to use a merge base here, which still is `beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee`. Any extra cherry-picks should be covered by the command listed below.